### PR TITLE
Cypress fixes:  'Create Project' timing issue and temporarily disable a11y check for 'aria-hidden-focus'

### PR DESCRIPTION
**Pull request by dtaylor113**
This PR addresses 2 Cypress issues:

1. Timing issue launching "Create Project" modal.   Cypress would immediatly click on the "Create Project" button to launch modal, then Project List page would finish loading and dismiss the modal!   Cypress now waits for Project List page to load before launching Create modal.

2. Continue to get `'aria-hidden-focus' accesibility violations` when launching modals.  #6910 was the first attempt to fix the violations; this PR is a continuation in the right direction, but not the complete solution, so this PR **temporarily disables the a11y check for aria-hidden-focus** until we correctly implement [aria-hidden elements do not contain focusable elements](https://dequeuniversity.com/rules/axe/3.4/aria-hidden-focus?application=axeAPI).  
Follow up [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1897008).

### CODE REVIEW

**Comment by rhamilto** [comment]
LGTM but will wait for other to weigh in.

**Comment by spadgett** [comment]
> Cypress would immediatly click on the "Create Project" button to launch modal, then Project List page would finish loading and dismiss the modal! Cypress now waits for Project List page to load before launching Create modal.

I don't disagree with waiting for the page to load, but the fact that the modal is dismissed is probably a bug in the console code. I wonder if that's something a user could ever see.

**Comment by dtaylor113** [comment]
> I don't disagree with waiting for the page to load, but the fact that the modal is dismissed is probably a bug in the console code. I wonder if that's something a user could ever see.

I don't think so, one of the commands was also `return`'ing a `cy...` which is also know to cause timing issues.

File: frontend/packages/integration-tests-cypress/views/list-page.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const listPage = {
       });
   },
   clickCreateYAMLbutton: () => {
-    return cy.byTestID('yaml-create').click();
+    cy.byTestID('yaml-create').click({ force: true });
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/views/list-page.ts:21]: Why is force needed?

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**